lib/matplotlib/widgets.py

```diff
@@ -266,94 +266,75 @@ class Slider(AxesWidget):
-
-      *valfmt*    : the format string for formatting the slider text
-
-      *label*     : a :class:`matplotlib.text.Text` instance
-                     for the slider label
-
-      *closedmin* : whether the slider is closed on the minimum
-
-      *closedmax* : whether the slider is closed on the maximum
-
-      *slidermin* : another slider - if not *None*, this slider must be
-                     greater than *slidermin*
-
-      *slidermax* : another slider - if not *None*, this slider must be
-                     less than *slidermax*
-
-      *dragging*  : allow for mouse dragging on slider
+    Create a slider from *valmin* to *valmax* in axes *ax*. For the slider to
+    remain responsive you must maintain a reference to it.
 
     Call :meth:`on_changed` to connect to the slider event
     """
     def __init__(self, ax, label, valmin, valmax, valinit=0.5, valfmt='%1.2f',
                  closedmin=True, closedmax=True, slidermin=None,
                  slidermax=None, dragging=True, **kwargs):
         """
-        Create a slider from *valmin* to *valmax* in axes *ax*.
-
-        Additional kwargs are passed on to ``self.poly`` which is the
-        :class:`matplotlib.patches.Rectangle` that draws the slider
-        knob.  See the :class:`matplotlib.patches.Rectangle` documentation for
-        valid property names (e.g., *facecolor*, *edgecolor*, *alpha*, ...).
-
         Parameters
         ----------
         ax : Axes
-            The Axes to put the slider in
+            The Axes to put the slider in.
 
         label : str
-            Slider label
+            Slider label.
 
         valmin : float
-            The minimum value of the slider
+            The minimum value of the slider.
 
         valmax : float
-            The maximum value of the slider
+            The maximum value of the slider.
 
-        valinit : float
-            The slider initial position
+        valinit : float, optional
+            The slider initial position.
+        Default: 0.5
 
-        label : str
-            The slider label
-
-        valfmt : str
-            Used to format the slider value, fprint format string
+        valfmt : str, optional
+            Used to format the slider value, fprint format string.
+            Default: '%1.2f'
 
-        closedmin : bool
-            Indicate whether the slider interval is closed on the bottom
+        closedmin : bool, optional
+            Indicate whether the slider interval is closed on the bottom.
+            Default: True
 
-        closedmax : bool
-            Indicate whether the slider interval is closed on the top
+        closedmax : bool, optional
+            Indicate whether the slider interval is closed on the top.
+            Default: True
 
-        slidermin : Slider or None
+        slidermin : Slider, optional
             Do not allow the current slider to have a value less than
-            `slidermin`
+            `slidermin.val`.
```

Comment:
Ah, I think I see what is going on here. This needs to be clearer. Perhaps, "Do not allow the slider to have a value less than the value of the other Slider `slidermin`."